### PR TITLE
[minor] Added gencfg-sls and gencfg-uds playbooks for using existing SLS nd UDS

### PR DESCRIPTION
**Pull request by shubhammalvankar**
- Added 2 playbooks **_gencfg-sls_** and **_gencfg-uds_** 
- Role variables for reading the tls cert file from the ENV variable were added
- Restructured _uds_contact_ role variable in the **_gencfg-uds_** role as the existing one wasn't working

### CODE REVIEW

**Review by whitfiea — CHANGES_REQUESTED** [review]
Some comments added, plus if new playbooks are being added then the docs for the playbooks need updating. Please update https://github.com/ibm-mas/ansible-devops/blob/master/docs/playbooks/dependencies.md with a description of what these playbooks do and what needs to be set.

File: ibm/mas_devops/roles/gencfg_sls/README.md [hunk 1]
```diff
@@ -24,19 +24,19 @@ Local directory to save the generated SlsCfg resource definition.  This can be u
...
 
-- Environment Variable: None
+- Environment Variable: `SLS_TLS_CERT_LOCAL_FILE`
 - Default Value: None
 
 ### slscfg_url
 Required. The URL of the LicenseService to be called when the Maximo Application Suite is registered with SLS.
 
-- Environment Variable: `SLSCFG_URL`
+- Environment Variable: `SLS_ENDPOINT_URL`
```
[the remark this answers is not in the document]
Comment (shubhammalvankar) [reply ibm/mas_devops/roles/gencfg_sls/README.md:33]: Okay I'll make this change

File: ibm/mas_devops/playbooks/dependencies/gencfg-sls.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+- hosts: localhost
+  any_errors_fatal: true
+  vars:
+    SLS_TLS_CERT_LOCAL_FILE: "{{ lookup('env', 'SLS_TLS_CERT_LOCAL_FILE_PATH') }}"
```
[the remark this answers is not in the document]
Comment (shubhammalvankar) [reply ibm/mas_devops/playbooks/dependencies/gencfg-sls.yml:5]: The lookup in the role is for a file, if I pass the variable as a path it doesn't seem to recognize it, hence I used a var as an ENV in the playbook which is passed onto the role as a file name

[the remark this answers is not in the document]
Comment (whitfiea) [reply ibm/mas_devops/playbooks/dependencies/gencfg-sls.yml:5]: Not sure I understand as both SLS_TLS_CERT_LOCAL_FILE and SLS_TLS_CERT_LOCAL_FILE_PATH are being set to the same value, so it shouldn't matter? Doing `SLS_TLS_CERT_LOCAL_FILE: "{{ lookup('env', 'SLS_TLS_CERT_LOCAL_FILE_PATH') }}" ` won't set SLS_TLS_CERT_LOCAL_FILE as the file contents.

[the remark this answers is not in the document]
Comment (shubhammalvankar) [reply ibm/mas_devops/playbooks/dependencies/gencfg-sls.yml:5]: Using `SLS_TLS_CERT_LOCAL_FILE: "{{ lookup('env', 'SLS_TLS_CERT_LOCAL_FILE_PATH') }}"` it looks up for the ENV SLS_TLS_CERT_LOCAL_FILE_PATH and uses that as a variable which is passed on to the role. 

Whereas directly using `slscfg_tls_crt: "{{ lookup('file', SLS_TLS_CERT_LOCAL_FILE) }}"`  considers `SLS_TLS_CERT_LOCAL_FILE` as the value rather then the actual value it holds. Hence it tries to lookup for file with the name `*/SLS_TLS_CERT_LOCAL_FILE` and not `*/sls.crt` [which is the file we need].

**Review by whitfiea — APPROVED** [review]

**Pull request merged**